build/rpms/kubelet.spec

```diff
@@ -6,13 +6,14 @@ Summary: Container Cluster Manager - Kubernetes Node Agent
 
 URL: https://kubernetes.io
 
+Requires: conntrack
```

Comment:
on *EL it's `conntrack-tools`

Reply:
I tested and depending on conntrack pulls it in on both ubuntu and rhel.

Reply:
```
$ sudo yum info conntrack
[sudo] password for tstclair:
Loaded plugins: fastestmirror, langpacks
Loading mirror speeds from cached hostfile
 * base: mirror.cs.uwp.edu
 * epel: mirror.steadfastnet.com
 * extras: mirror.cs.uwp.edu
 * ovirt-4.2: www.gtlib.gatech.edu
 * ovirt-4.2-epel: mirror.steadfastnet.com
 * updates: mirror.steadfastnet.com
Error: No matching Packages to list
```

Reply:
```
$ sudo yum info conntrack-tools
Loaded plugins: fastestmirror, langpacks
Loading mirror speeds from cached hostfile
 * base: mirror.cs.uwp.edu
 * epel: mirror.steadfastnet.com
 * extras: mirror.cs.uwp.edu
 * ovirt-4.2: www.gtlib.gatech.edu
 * ovirt-4.2-epel: mirror.steadfastnet.com
 * updates: mirror.steadfastnet.com
Available Packages
Name        : conntrack-tools
Arch        : x86_64
Version     : 1.4.4
Release     : 3.el7_3
Size        : 186 k
Repo        : base/7/x86_64
Summary     : Manipulate netfilter connection tracking table and run High Availability
URL         : http://netfilter.org
License     : GPLv2
Description : With conntrack-tools you can setup a High Availability cluster and
            : synchronize conntrack state between multiple firewalls.
            :
            : The conntrack-tools package contains two programs:
            : - conntrack: the command line interface to interact with the connection
            :              tracking system.
            : - conntrackd: the connection tracking userspace daemon that can be used to
            :               deploy highly available GNU/Linux firewalls and collect
            :               statistics of the firewall use.
            :
            : conntrack is used to search, list, inspect and maintain the netfilter
            : connection tracking subsystem of the Linux kernel.
            : Using conntrack, you can dump a list of all (or a filtered selection  of)
            : currently tracked connections, delete connections from the state table,
            : and even add new ones.
            : In addition, you can also monitor connection tracking events, e.g.
            : show an event message (one line) per newly established connection.
```

Reply:
kk updating

Reply:
Per our conversation leaving this alone as the conntrack-tools package "provides" conntrack.